oxd-server/src/test/java/org/xdi/oxd/server/HealthCheckTest.java

```diff
@@ -0,0 +1,27 @@
+package org.xdi.oxd.server;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import org.xdi.oxd.client.ClientInterface;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNotNull;
+
+public class HealthCheckTest {
+
+    private static final Logger LOG = LoggerFactory.getLogger(HealthCheckTest.class);

```

Comment:
please remove logger. It seems we don't need it. TestNG already reports parameters, no need to duplicate this information.

![image](https://user-images.githubusercontent.com/2013070/53883512-d7cac780-4021-11e9-850b-39829ead238d.png)